pkg/validation/validation-rule.md

```diff
@@ -60,7 +63,3 @@ Common rules for all components types:
 ### projects
 - if more than one remote is configured, a checkout remote is mandatory
 - if checkout remote is mentioned, validate it against the starter project remote configured map
-
-### Architectures
```

Comment:
Could you please elaborate why you removes it?

Reply:
The file `validation-rule.md ` only details the validation done explicitly in `pkg/validation` which is not covered by the schema. 

Currently, arch is covered by the schema enum https://github.com/devfile/api/blob/main/pkg/devfile/header.go#L20-L29